src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -205,11 +205,9 @@ struct CharacteristicEvolution {
       const Parallel::CProxy_ConstGlobalCache<Metavariables>&
           global_cache) noexcept {
     auto& local_cache = *(global_cache.ckLocalBranch());
-    if (next_phase == Metavariables::Phase::Evolve) {
-      Parallel::get_parallel_component<CharacteristicEvolution<Metavariables>>(
-          local_cache)
-          .start_phase(next_phase);
-    }
+    Parallel::get_parallel_component<CharacteristicEvolution<Metavariables>>(
```

Comment:
why was this change made?

Reply:
upon @nilsleiffischer 's request to remove the `if`